### ISSUE TITLE
unnecessary loop in com.s3auth.hosts.DynamoHosts.Wrap#size()

### ISSUE THREAD

**Issue by simonjenga** [yegor256/s3auth#315]
Why do we need the ```while``` loop in the current implementation of ```DynamoHosts.Wrap#size()```.

```
@Override
public int size() {
	final Iterator<?> iterator = this.iterator();
	int size = 0;
	while (iterator.hasNext()) {
		iterator.next();
		++size;
	}
	return size;
}
```

Can't we just ```return this.domains.size();```

**Comment by davvd** [comment]
@yegor256 please pay attention to this issue ([par.21](http://www.teamed.io/policy.html#21))

**Comment by yegor256** [comment]
@simonjenga fixed in e2ab63a thanks!
